client/src/views/all-games/components/FeedbackForm.tsx

```diff
@@ -73,11 +74,11 @@ export const FeedbackForm = ({ game }: Props): ReactElement => {
   );
 };
 
-const FeedbackTextarea = styled.textarea`
-  width: 95%;
-  border: 1px solid black;
-  resize: none;
-  overflow: auto;
+const FeedbackTextarea = styled(Textarea)`
+  width: 100%;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
```

Comment:
Miun käsityksen mukaan näitä selainkohtaisia propertyja ei pitäs tarvita. `styled-components` huolehtii niistä.

> Automatic vendor prefixing: write your CSS to the current standard and let styled-components handle the rest.

https://styled-components.com/docs/basics#motivation